### DISCUSSION TITLE
请问点击表格行高亮的效果，可以设置一个支持取消被点击单元格加边框的属性吗？

### DISCUSSION THREAD

**Discussion by xixia7** [Happy-Coding-Clans/vue-easytable#462] [Ideas]
点击表格行高亮的时候，不想要被点击的单元格加上边框，请问可以设置一个属性控制吗 :)

**Comment by huangshuwei** [comment]
当然可以，禁用单元格选择即可。具体请查看文档：https://happy-coding-clans.github.io/vue-easytable/#/zh/doc/table/cell-selection?anchor=jin-yong-dan-yuan-ge-xuan-ze
